vhdbuilder/packer/init-variables.sh

```diff
@@ -326,19 +335,35 @@ if [ "$OS_TYPE" == "Windows" ]; then
 				--location $AZURE_LOCATION \
 				--hyper-v-generation $HYPERV_GENERATION \
 				--os-type ${OS_TYPE}
+			
+			# Showing the created managed image from imported image
+			id=$(az image show --name ${IMPORTED_IMAGE_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id)
```

Comment:
Do you use these code to show the created image, definition and version for debug? Suggest to remove it for better perf

Reply:
Removed.